### PR TITLE
Submitting blank last words while succumbing doesn't stop you from dying 

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request
Prefaces the succumb prompt with a "Are you sure you want to succumb?" After you click yes on this you will die no matter how the last words box is finished (Summited black, canceled, or closed with the X)

## Why It's Good For The Game
Its annoying to type out some meaningless last words when you are just sat there alone and dying.

## Changelog

:cl: itseasytosee
fix: Leaving the last words box empty in the succumb prompt will not stop you from dying.
/:cl:

### CODE REVIEW

File: code/_onclick/hud/alert.dm [hunk 1]
```diff
@@ -419,8 +419,14 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 		return
 
 	var/mob/living/living_owner = owner
+	var/confirm = tgui_alert(usr,"Are you sure you want to succumb to death?", "Goodnight, Sweet Prince", list("Yes", "No"))
+
+	if(!confirm)
+		return
```
Comment (MrMelbert) [inline code/_onclick/hud/alert.dm:425]:
```suggestion
	if(!confirm || !CAN_SUCCUMB(living_owner))
		return
```

File: code/_onclick/hud/alert.dm [hunk 2]
```diff
@@ -419,8 +419,14 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 		return
 
 	var/mob/living/living_owner = owner
+	var/confirm = tgui_alert(usr,"Are you sure you want to succumb to death?", "Goodnight, Sweet Prince", list("Yes", "No"))
```
Comment (MrMelbert) [inline code/_onclick/hud/alert.dm:422]:
```suggestion
	var/confirm = tgui_alert(usr ,"Are you sure you want to succumb to death?", "Goodnight, Sweet Prince", list("Yes", "No"))
```

**Comment by jlsnow301** [comment]
frick I didn't even see this. well, it's working as intended with my PR, let me know which is preferred. I am willing to close #66653

**Comment by MrMelbert** [comment]
@jlsnow301 I prefer this one over the last tbh

In your PR if you click "succumb" and want to stop succumbing you can't
In this pr there's a second input that says "are you sure" that you can cancel out of

File: code/_onclick/hud/alert.dm [hunk 3]
```diff
@@ -419,8 +419,14 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 		return
 
 	var/mob/living/living_owner = owner
+	var/confirm = tgui_alert(usr ,"Are you sure you want to succumb to death?", "Goodnight, Sweet Prince", list("Yes", "No"))
```
Comment (MrMelbert) [inline code/_onclick/hud/alert.dm:422]:
```suggestion
	var/confirm = tgui_alert(usr, "Are you sure you want to succumb to death?", "Goodnight, Sweet Prince", list("Yes", "No"))
```
wrong side

File: code/_onclick/hud/alert.dm [hunk 4]
```diff
@@ -419,8 +419,14 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
...
+
+	if(!confirm || !CAN_SUCCUMB(living_owner))
```
Comment (MrMelbert) [inline code/_onclick/hud/alert.dm:424]:
```suggestion
	if(confirm == "No" || !CAN_SUCCUMB(living_owner))
```

**Comment by jlsnow301** [comment]
> In your PR, if you click "succumb" and want to stop succumbing, you can't

you can click cancel or close the window to stop succumbing
this is live and unchanged with my pr

**Comment by itseasytosee** [comment]
I'll leave this open and let maintainers decide which one they want.

**Comment by MrMelbert** [comment]
> > In your PR, if you click "succumb" and want to stop succumbing, you can't
> 
> you can click cancel or close the window to stop succumbing this is live and unchanged with my pr

guh? The more you know

**Pull request closed without merging**